bindings/pyroot_experimental/PyROOT/test/tcomplex_radd_rsub.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import ROOT
+from ROOT import TComplex
+
+class TestTComplexMethods(unittest.TestCase):
```

Comment:
I would call the test `TestTComplexOperators`, since we could have other pythonisations for other methods in the future.